apps/api/mvm.lock

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
What's this file for?

Reply:
This file is generated by that mvm package. It's keeping current monorepo package versions in the client apps. File is updated on pre-commit hook. This is needed for released to properly detect changes and bump a version